Update evaluate-attack-surface-reduction.md (#4759)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/evaluate-attack-surface-reduction.md

Learn how to evaluate attack surface reduction rules by enabling audit mode to test the feature directly in your organization.

> > [!Warning]

Reviewer: ```suggestion
> [!WARNING]
```

> > Enabling Attack Service Reduction rules on Windows Server 2016 may lead to unexpected results and impact server performance. It is not recommended to enable or deploy Attack Surface Reduction rules to unsupported platforms.

Reviewer: ```suggestion
> Enabling attack service reduction rules on Windows Server 2016 may lead to unexpected results and impact server performance. We do not recommend enabling or deploying attack surface reduction rules to unsupported platforms.
```
